tests/src/JIT/Regression/JitBlue/GitHub_5047/project.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
We do use individual project.json files for tests that have dependencies that are not included in the common projects.